CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+## Staging
+
+- Prescriptive
+  - Exploratory
+  - Draft
+  - Final
+- Descriptive
+  - Draft
+  - Final
+
+Prescriptive specifications are intended to describe future 
+implementations or, in some cases, changes to existing implementations.
+
+Descriptive specifications describe existing behavior. In many cases
+these specifications are not intended to drive new implementations and
+are only written to understand existing behaviors.
+
```

Comment:
I suggest we also say how to identify the stage of a spec, something like:

> Specification documents should include a descriptor in their heading that indicates their status in the staging process. e.g. "Prescriptive - Draft", or "Descriptive - Final".

Then we can go about amending what's in here.